src/Vault.sol

```diff
@@ -913,11 +913,13 @@ contract Vault is IVault {
 
         uint256 k;
         uint256 erc1155StoredLength = erc1155Stored.length;
+        uint256 cacheId;
```

Comment:
this is actually 5-10 gas more expensive during liquidations
![image](https://user-images.githubusercontent.com/104004836/221385314-4362da04-c037-4ac0-9001-b132dbf59354.png)